setup.py

```diff
@@ -5,8 +5,14 @@
 
 # Get a list of scripts to install
 scripts = []
-for root, _, filenames in os.walk('scripts'):
+for root, dirnames, filenames in os.walk('scripts'):
```

Comment:
Interesting, I wonder if that was different at some time in the past. The other examples I was reading through were very explicit about calling it out.